packages/packagers/markdown-html/src/MarkdownHtmlTransformer.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import assert from 'assert';
+import {Packager} from '@parcel/plugin';
+import {replaceURLReferences} from '@parcel/utils';
+import Mustache from 'mustache';
+
+export default new Packager({
```

Comment:
@mischnic I'm not entirely sure how you saw this? How would this ever output html with the html extension?

Reply:
Also there's no way I'm aware of the get the result of another packager from within a packager. As html assets get replaced in packaging, mainly inline assets?

Reply:
I somehow mixed up transformer and packager and thought that a packager could change the type 🙈 

Maybe the named pipeline system should be extended to packagers as well?

Reply:
Not sure @devongovett any ideas?